src/components/CertificateDropZone/CertificateDropZoneContainer.js

```diff
@@ -42,26 +53,44 @@ class CertificateDropZoneContainer extends Component {
     this.setState({ fileError: true });
   }
 
+  toggleQrReaderVisible() {
+    this.setState({ qrReaderVisible: !this.state.qrReaderVisible });
+  }
+
   resetData() {
     this.props.resetData();
   }
 
   render() {
-    return (
-      <CertificateDropZone
-        document={this.props.document}
-        fileError={this.state.fileError}
-        handleCertificateChange={this.handleCertificateChange}
-        handleFileError={this.handleFileError}
-        verifying={this.props.verifying}
-        issuerIdentityStatus={this.props.issuerIdentityStatus}
-        hashStatus={this.props.hashStatus}
-        issuedStatus={this.props.issuedStatus}
-        notRevokedStatus={this.props.notRevokedStatus}
-        verificationStatus={this.props.verificationStatus}
-        resetData={this.resetData.bind(this)}
-        storeStatus={this.props.storeStatus}
-      />
+    return this.state.qrReaderVisible ? (
+      <>
+        <QrReader handleQrScanned={this.handleQrScanned} />
+        <button
+          type="button"
+          onClick={this.toggleQrReaderVisible}
+          className={`pointer ${css.btn} ${css["disable-btn"]}`}
+        >
+          Disable
+        </button>
+      </>
+    ) : (
+      <>
```

Comment:
not needed fragment also. Only a single child is there.

Reply:
Removed!